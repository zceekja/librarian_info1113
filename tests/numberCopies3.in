add collection book1.csv
add collection book2.csv
number copies