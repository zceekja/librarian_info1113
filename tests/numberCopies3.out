user: 17 books successfully added.

user: 17 books successfully added.

user: Aeneid (Virgil): 2
Batman Volume 1 (Bob Kane): 2
Biggles in the Battle (W.E. Johns): 2
Halloween (Jamie Lee Curtis): 2
Harry Potter 1 (J.K. Rowling): 1
Harry Potter 10 (J.K. Rowling): 1
Harry Potter 11 (J.K. Rowling): 1
Harry Potter 12 (J.K. Rowling): 1
Harry Potter 2 (J.K. Rowling): 1
Harry Potter 3 (J.K. Rowling): 1
Harry Potter 4 (J.K. Rowling): 1
Harry Potter 5 (J.K. Rowling): 1
Harry Potter 6 (J.K. Rowling): 1
Harry Potter 7 (J.K. Rowling): 1
Harry Potter 8 (J.K. Rowling): 1
Harry Potter 9 (J.K. Rowling): 1
So Long and Thanks for all the Fish (Douglas Adams): 2
The Hitchhiker's Guide to the Galaxy (Douglas Adams): 2
The Hobbit (J.R.R. Tolkien): 2
The Iliad (Homer): 2
The Odyssey (Homer): 2
To Kill a Mockingbird (Harper Lee): 2
Wasteland (T.S. Eliot): 2

user: 